Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -294,7 +294,10 @@ void CtapAuthenticator::getAssertion()
         if (!m_batches.size())
             return continueGetAssertionAfterCheckAllowCredentials();
         m_currentBatch = 0;
-        Vector<uint8_t> cborCmd = encodeSilentGetAssertion(options.rpId, requestData().hash, m_batches[m_currentBatch], std::nullopt);
+        std::optional<PinParameters> pinParameters;
+        if (!m_pinAuth.isEmpty())
+            pinParameters = PinParameters { pin::kProtocolVersion, m_pinAuth };
```

Comment:
nit, to benefit from in-place construction
```suggestion
            pinParameters.emplace(pin::kProtocolVersion, m_pinAuth);
```

Reply:
I will do this before landing, but I am opting not to use EWS resources on it now.

Reply:
Actually going to do this in a separate change to be safe: https://bugs.webkit.org/show_bug.cgi?id=295045